helpers/polizasHelper/polizashelper.go

```diff
@@ -0,0 +1,260 @@
+	"github.com/udistrital/arka_mid/models"
+	"github.com/udistrital/utils_oas/formatdata"
+	"github.com/udistrital/utils_oas/request"
+)
+
+// GetElementosPoliza Obtiene todos los elementos que necesiten poliza
+//
+func GetElementosPoliza(offset int, limit int, fields []string, order []string, query map[string]string, sortby []string) (ElementosPoliza *[]models.Elemento, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "GetElementosPoliza - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		ActasIdsEntradas []int
+		SubgrupoPoliza   []int
+	)
+
+	// Trae las ActasId que son entradas en movimientoservice.movimiento
+	if Elementos, err := GetIdsActasEntrada(); err != nil {
+		return nil, err
+	} else {
+		ActasIdsEntradas = Elementos
+	}
+
+	// Obtiene IdDetalleSubgrupo de los tipos de bien en catalogoelemento.detallesubgrupo
+	if idsubgrupo, err := GetSubgruposPoliza(); err != nil {
+		return nil, err
+	} else {
+		SubgrupoPoliza = idsubgrupo
+	}
+
+	// Respuesta de los Elementos que necesiten poliza y pudiendo filtrar
+	if RElementosPoliza, err := GetElementosPolizas(ActasIdsEntradas, SubgrupoPoliza, limit, offset, fields, order, query, sortby); err != nil {
+		return nil, err
+	} else {
+		ElementosPoliza = RElementosPoliza
+	}
+
+	return
+}
+
+// GetElementosEntrada obtiene todos las ActasId que sean entradas (P8)
+//
+func GetIdsActasEntrada() (IdsActasEntradas []int, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "GetIdsActasEntrada - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		RespuestaAPI     interface{}
+		Movimientos      []models.Movimiento
+		ActasIdsEntradas []int
+	)
+
+	//Traer el detalle de las entradas (json)
+	params := url.Values{}
+	params.Add("query", "Activo:true,Detalle__contains:consecutivo\": \"P")
+	params.Add("fields", "Detalle")
+	params.Add("limit", "-1")
+	urlRespuestaAPI := "http://" + beego.AppConfig.String("movimientosArkaService") + "movimiento?" + params.Encode()
+	if _, err := request.GetJsonTest(urlRespuestaAPI, &RespuestaAPI); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetIdsActasEntradas - request.GetJsonTest(urlRespuestaAPI, &RespuestaAPI)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	// Asignar a Movimientos RespuestaAPI
+	if err := formatdata.FillStruct(&RespuestaAPI, &Movimientos); err != nil {
+		logs.Error(err)
+	}
+
+	//Itera los Movimientos para poder guardar solo las actas de recibido
+	for _, movimiento := range Movimientos {
+		var detalle map[string]interface{}
+		if err := json.Unmarshal([]byte(movimiento.Detalle), &detalle); err != nil {
+			logs.Error(err)
```

Comment:
Revisar: Tal como quedó, si hay error, "no hay problema", únicamente se imprime en consola, ¿esto es correcto?
De ser así, no sería un error, por mucho una advertencia, cambiar:
```suggestion
			logs.Warn(err)
```

En caso contrario, se debería mantener tal como está el logs.Error` y luego armar y retornar un error formateado